### ISSUE TITLE
TypeError: Unknown keyword argument "StandardName" in MetaDataVersion when using -dxp with Define-XML v2.1

### ISSUE THREAD

**Issue by AltleoB2B** [cdisc-org/cdisc-rules-engine#1826]
### Standard

CDISC SEND

### Reference Rule ID(s)

N/A

### Conformance Rule ID(s) (if published in CORE)

CORE-

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### CLI Command Used (if applicable)

python core.py -s SDTM -v 3.4 -d /path/to/datasets

### Rule YAML

```YAML
Authorities: 
  - Organization: 
    Standards:
      - Name: 
        References:
          - Citations:
              - Cited Guidance: 
                Document: 
                Item: 
                Section: 
            Origin: 
            Rule Identifier: 
              Id: 
              Version: 
            Version: 
        Version:
Check:
  all:
    - name: 
      operator: 
      value: 
Core:
  Id: 
  Status: 
  Version: '1'
Description: 
Executability: Fully Executable
Match Datasets:
  - Keys:
      - 
    Name: 
Operations:
  - id: 
    operator: 
Outcome:
  Message: 
  Output Variables:
    -
Rule Type: 
Scope:
Sensitivity:
```

### Attach any sample test data file(s)

NA

### Attach any output report and/or log file(s)

NA

### Expected output

NA

## Description
When passing a Define-XML v2.1 file via the `-dxp` flag, CORE crashes with a 
TypeError from odmlib.

## Error
TypeError: Unknown keyword argument StandardName in MetaDataVersion

## Steps to reproduce
1. Generate a valid Define-XML v2.1 file with def:StandardName and 
   def:StandardVersion attributes on the MetaDataVersion element
2. Run: core.exe validate -s sendig -v 3-1-1 -d <xpt-dir> -dxp <define.xml> -of JSON
3. CORE crashes immediately with the above TypeError

## Root cause
odmlib does not accept def:StandardName and def:StandardVersion attributes 
on <MetaDataVersion>. These are valid Define-XML v2.1 attributes per the 
CDISC Define-XML 2.1 specification.

## Workaround
Strip def:StandardName and def:StandardVersion from define.xml before passing 
to CORE via -dxp. This avoids the crash but means CORE cannot verify the 
StandardName/StandardVersion metadata.

## Environment
- CORE version: 0.16.0
- Define-XML version: 2.1.0
- OS: Windows Server 2019

## Expected behaviour
CORE should accept all valid Define-XML v2.1 attributes when processing 
via the -dxp flag.

**Comment by OGarcia11** [comment]
Hi @AltleoB2B, https://github.com/swhume/odmlib is the repo where to address this issue. Please create a ticket there.
